planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/test/src/test_occlusion_spot_utils.cpp

```diff
@@ -155,37 +158,38 @@ TEST(createPossibleCollisionBehindParkedVehicle, TooManyPathPointsAndObstacles)
             << " 1 -   |   |   |   |                    \n"
             << " 0 | 1 | 2 | 3 | 4 | \n";
 
-  autoware_perception_msgs::msg::DynamicObjectArray obj_arr;
-  autoware_perception_msgs::msg::DynamicObject obj;
+  autoware_auto_perception_msgs::msg::PredictedObjects obj_arr;
+  autoware_auto_perception_msgs::msg::PredictedObject obj;
   obj.shape.dimensions.x = 0.0;
   obj.shape.dimensions.y = 0.0;
   tf2::Quaternion q;
-  obj.state.pose_covariance.pose.orientation = tf2::toMsg(q);
-  obj.state.orientation_reliable = true;
-  obj.state.twist_reliable = true;
-  obj.state.twist_covariance.twist.linear.x = 0;
+  obj.kinematics.initial_pose_with_covariance.pose.orientation = tf2::toMsg(q);
+  // TODO(murooka) reliable
+  // obj.kinematics.orientation_reliable = true;
+  // obj.kinematics.twist_reliable = true;
```

Comment:
How can I deal with reliable of predicted object kinematics since this message doesn't have.
then remove TODO(murooka)

Reply:
@yukkysaito

Reply:
The orientation_reliable has been removed because the policy is to make sure that sign(front-back orientation) is fixed in perception as much as possible.
However, there is a possibility that it may be wrong, so it may be better to have the planner code that sign may be wrong.

Reply:
You mean, so far it's ok not to think about realiable right now and I will just remove realiable, right?
Then as a future task, we should deal with the sign of orientation in the behavior?
@yukkysaito

Reply:
@takayuki5168 
確実にやり取りしたいので日本語にします
今までのorientation reliableは前後方向がわかっていないよ！というフラグだったと思いますが、今回はperceptionの出力としては前後方向も確定して出してください！という方針になっています。
そのためrealiable自体msgにないと思います。
もちろんperception側で前後方向は確定させるのですが、やはりミスがでる可能性があります。もし今でのorientation reliableを使ってplanner側でも前後方向が間違ってい場合の処理が書いてあったのであれば、今回はすべてorientation reliable==falseという前提で組んでもらうのが良いと思います

Reply:
@yukke42 I got it.
Now reliable is only used in unit tests (It seems the unit tests just assigned true to reliable), so we can remove it without any modification or future task.

Reply:
https://github.com/tier4/AutowareArchitectureProposal.iv/pull/605/commits/efaafd118c4e5dc327724dcd6219c21b9d3017c0
Done